### PR TITLE
Fixes #1382 : Simplifies the user-guide component

### LINKED ISSUE #1382: simplifying the user-guide component
**I'm submitting a ...**
- [ ] bug report
- [X] feature request

There are a lot of `raised segments` and according to last meetings the extra boxes are not required.
Therefore it must be removed.

**I am sending a PR for the same**

### PR DESCRIPTION
**Pull request by gabru-md**
Fixes #1382 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Removes extra boxes from user-guide component.

Cheers!

### CODE REVIEW

File: frontend/app/templates/components/user-component/user-guide.hbs [hunk 1]
```diff
@@ -5,7 +5,7 @@
     </div>
     <div class="ui segments">
```
Comment (yashLadha) [inline frontend/app/templates/components/user-component/user-guide.hbs]: Can be changed to basic segments every where?

File: frontend/app/templates/components/user-component/user-guide.hbs [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
         <div class="ui segment">
-            <div class="ui raised segment">
+            <div class="ui">
```
Comment (yashLadha) [inline frontend/app/templates/components/user-component/user-guide.hbs]: Can be changed to 
```html
<div class='ui basic segment'>
</div>
```

Comment (gabru-md) [reply frontend/app/templates/components/user-component/user-guide.hbs]: with basic segment
shif


with this PR
guuide


the current changes looks more even than the `basic segment` would look. you can see that the titles are a bit shifted towards right.

Comment (yashLadha) [reply frontend/app/templates/components/user-component/user-guide.hbs]: Fine then

**Pull request merged**